std/root.ln

```diff
@@ -611,6 +617,19 @@ export fn newHashMap(firstKey: Hashable, firstVal: any): HashMap<Hashable, any>
   return hm.set(firstKey, firstVal)
 }
 
+export fn asHashMap(kva: Array<KeyVal<Hashable, any>>) {
```

Comment:
If you can think of a better name I'm all ears. :)

Reply:
I prefer `toHashMap`

Reply:
Got it. Changing.